cmd/vicadmin/vicadm.go

```diff
@@ -46,15 +46,20 @@ import (
 	"github.com/vmware/govmomi/vim25/types"
 
 	"github.com/vmware/vic/lib/metadata"
+	"github.com/vmware/vic/lib/pprof"
 	"github.com/vmware/vic/pkg/trace"
 	"github.com/vmware/vic/pkg/vsphere/compute"
 	"github.com/vmware/vic/pkg/vsphere/extraconfig"
 	"github.com/vmware/vic/pkg/vsphere/session"
 	"github.com/vmware/vic/pkg/vsphere/vm"
 )
 
+const (
+	timeout = time.Duration(2 * time.Second)
```

Comment:
Oh this is not how we download logs from ESXi and used only for pprof logs? If that's the case yeah, it doesn't matter that much...